pkg/setting/setting.go

```diff
@@ -757,6 +761,14 @@ func (cfg *Cfg) validateStaticRootPath() error {
 func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	setHomePath(args)
 
+	// Fix for missing IANA db on Windows
+	_, zoneInfoSet := os.LookupEnv(zoneInfo)
+	if runtime.GOOS == "windows" && !zoneInfoSet {
+		if err := os.Setenv(zoneInfo, HomePath+"/tools/zoneinfo.zip"); err != nil {
```

Comment:
BTW, shouldn't you instead use the following to get the right path separator on Windows?

```suggestion
		if err := os.Setenv(zoneInfo, filepath.Join(HomePath, "tools", "zoneinfo.zip")); err != nil {
```

Reply:
yes, probably I should. Thanks you. Though somehow the concatenated strings version worked on Win as well 🤔